luigi/worker.py

```diff
@@ -300,6 +300,10 @@ class worker(Config):
                                          config_path=dict(section='core', name='retry-external-tasks'),
                                          description='If true, incomplete external tasks will be '
                                          'retested for completion while Luigi is running.')
+    install_shutdown_handler = BoolParameter(default=True,
+                                             config_path=dict(section='core', name='worker-install-shutdown-handler'),
```

Comment:
Remove this line. `config_path` should actually be called `old_deprecated_config_path` so to say.